.vscode/settings.json

```diff
@@ -10,8 +10,7 @@
   },
   "typescript.tsdk": "./node_modules/typescript/lib",
   "editor.tabSize": 2, // we want to use the TS server from our node_modules folder to control its version,
-  "editor.formatOnSave": true,
```

Comment:
This looks like a spurious change?

Reply:
The current package.json doesn't line up with the format on save option, so I turned it off so the diff wouldn't be unreadable. If you're happy with the current diff I'll reenable this setting and then reformat it?